bagisto/bagisto
# Responsive issue in user Profile - Velocity (Fixed)

## Issue Reference
#5259
## Description
Responsive issue in user Profile - Velocity (Fixed)

## How To Test This?
Pull the latest code & login with customer & visit profile page and check the responsiveness in desktop & mobile devices

## Documentation
Not At All

Update Velocity.css File along with updated app.scss file
